auto/configure.py

```diff
@@ -207,7 +207,7 @@ def generate_user_auth(self, password):
     def generate_cega_mq_auth(self, cega_pwd):
         """Generate CEGA MQ auth."""
         generated_secret = cega_pwd if cega_pwd else self._generate_secret(32)
-        cega_defs_mq = """{{"rabbit_version":"3.6.11",\r\n     "users":[{{"name":"lega",
+        cega_defs_mq = """{{"rabbit_version":"3.6.14",\r\n     "users":[{{"name":"lega",
```

Comment:
Don't set subversion, any 3.6 subversion >= 14 will work. Currently using latest == 16.

Reply:
was trying to match what we have in current LocalEGA. but you are right, changed.